eq-author/src/App/settings/ThemesPage.js

```diff
@@ -42,6 +43,25 @@ const StyledPanel = styled.div`
   padding: 1.3em;
 `;
 
+const HorizontalSeparator = styled.hr`
+  border: 0;
+  border-top: 0.0625em solid ${colors.lightMediumGrey};
+  margin: 1.5em 0;
+`;
+
+const themes = [
+  {
+    title: "GB theme",
+    defaultOpen: true,
+  },
+  { title: "NI theme" },
+  { title: "COVID theme" },
+  { title: "EPE theme" },
+  { title: "EPE NI theme" },
+  { title: "UKIS theme" },
+  { title: "UKIS NI theme" },
+];
+
```

Comment:
Not here specifically but on line 68, there's a hook called `useRouteMatch` that's used to get the params. Could we use useParams instead? We are only getting the params from it